jekyll/_cci2/style/formatting.adoc

```diff
@@ -4,19 +4,34 @@
 :toc: macro
 :toc-title:
 
+[#styling-text]
 == Styling text
 
 * Use **bold** for GUI menu options and button text. For example:
 ** Select **Organization Settings** from the sidebar
 ** Select **VCS** in the **Project Settings** window
 ** Select **Projects** from the sidebar and click **Set Up Project** for the project you want to follow
+* Also use **bold** to highlight key words in intro paragraphs. This helps with SEO.
 
 * Use italics for technical terms, for example:
 ** _Continuous Integration_
 
-== Using notes, tips, important information, cautions and warnings
+[#using-notes-tips-cautions-warnings]
+== Using notes, tips, cautions, and warnings
 
-To add an admonition (in asciidoc) - note, tip, important information, caution, warning – use the following syntax:
+When writing in asciidoc the following _admonitions_ are available: note, tip, important, caution, warning. 
+
+[#when-to-use-each-type]
+=== When to use each type
+
+* `note`: Use to add some supplementary information to a section or point that could benefit from some highlighting to draw the reader's attention. 
+* `tip`: Use to add some guidance on how to carry out a step or how to act on some advice contained within the section.
+* `important`: Not used in CircleCI docs. 
```

Comment:
What does this mean? We shouldn't use it at all? That we are referencing something that is outside of CircleCI's control? And if it's that we just don't use it, my question is why?